framework/src/org/checkerframework/common/value/ValueVisitor.java

```diff
@@ -49,6 +53,29 @@ private long getIntLiteralValue(LiteralTree exp) {
         }
     }
 
+    /**
+     * ValueVisitor overrides this method so that it does not have to check variables annotated with
+     * the {@link IntRangeFromPositive} annotation. This annotation is only introduced by the Index
+     * Checker's {@link org.checkerframework.checker.index.qual.Positive} annotation. It is safe to
+     * defer checking of these values to the Index Checker because this is only introduced for
+     * explicitly-written {@link org.checkerframework.checker.index.qual.Positive} annotations,
+     * which must be checked by the Lower Bound Checker.
+     *
+     * @param varType the annotated type of the lvalue (usually a variable)
+     * @param valueExp the AST node for the rvalue (the new value)
+     * @param errorKey the error message to use if the check fails (must be a compiler message key,
+     */
+    @Override
+    protected void commonAssignmentCheck(
+            AnnotatedTypeMirror varType,
+            ExpressionTree valueExp,
+            /*@CompilerMessageKey*/ String errorKey) {
+        AnnotationMirror anm = varType.getAnnotationInHierarchy(atypeFactory.UNKNOWNVAL);
```

Comment:
This doesn't work if the `@Positive` annotation is any where besides `@Positive int`.  For example, it misses `@Positive int []`.  You can us a `org.checkerframework.framework.type.visitor.SimpleAnnotatedTypeScanner` to change `@IntRangeFromPositive` to `@UnknownVal` in all annotation locations.